build.gradle

```
@@ -5,7 +5,7 @@ plugins {
 }
 
 group 'com.github.hmcts'
-version '1.15.3'
+version '1.15.4'
```

I would suggest making this 1.16.0 instead since we're supporting a new event
```suggestion
version '1.16.0'
```